sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/PartitionReceiver.cs

```diff
@@ -251,12 +262,37 @@ protected PartitionReceiver()
         ///
         /// <returns>The set of information for the associated partition under the Event Hub this client is associated with.</returns>
         ///
-        public async virtual Task<PartitionProperties> GetPartitionPropertiesAsync(CancellationToken cancellationToken = default)
+        public virtual async Task<PartitionProperties> GetPartitionPropertiesAsync(CancellationToken cancellationToken = default)
         {
             Argument.AssertNotClosed(IsClosed, nameof(PartitionReceiver));
             return await Connection.GetPartitionPropertiesAsync(PartitionId, RetryPolicy, cancellationToken).ConfigureAwait(false);
         }
 
+        /// <summary>
+        ///   Receives a batch of <see cref="EventData" /> from the Event Hub partition this client is associated with.
+        /// </summary>
+        ///
+        /// <param name="maximumEventCount">The maximum number of messages to receive in this batch.</param>
+        /// <param name="maximumWaitTime">The maximum amount of time to wait to build up the requested message count for the batch; if not specified, the default wait time specified by the options when the client was created will be used.</param>
+        /// <param name="cancellationToken">An optional <see cref="CancellationToken"/> instance to signal the request to cancel the operation.</param>
+        ///
+        /// <returns>The batch of <see cref="EventData" /> from the Event Hub partition this client is associated with.  If no events are present, an empty enumerable is returned.</returns>
+        ///
+        public virtual Task<IEnumerable<EventData>> ReceiveBatchAsync(int maximumEventCount,
```

Comment:
You're still planning the override for this that does not take a maximum wait time, yeah?

Reply:
Correct.